### DISCUSSION TITLE
Bookmarks location

### DISCUSSION THREAD

**Discussion by manos00** [minbrowser/min#1822] [Q&A] [answered]
Is the database where bookmarks are saved encrypted? If not, does anybody know where this database is located? 
Else I'd like to know if you can somehow automate exporting bookmarks to an html file.
I want to use the information in a script.
Thanks in advance.

**Comment by PalmerAL** [comment, marked as answer]
> Is the database where bookmarks are saved encrypted

No.

> does anybody know where this database is located

Mac: ~/Library/Application Support/Min/IndexedDB/file__0.indexeddb.leveldb
Linux: ~/.config/Min/IndexedDB/...
Windows: C:\Users\<User>\AppData\Roaming\Min\IndexedDB\...

> Else I'd like to know if you can somehow automate exporting bookmarks to an html file.

Not yet: https://github.com/minbrowser/min/discussions/1767. But you can export manually by typing `!exportbookmarks` in the searchbar.
